docs/releases/v2022.1.4.rst

```diff
@@ -0,0 +1,126 @@
+Gluon 2022.1.4
+==============
+
+Added hardware support
+----------------------
+
+ath79-generic
+~~~~~~~~~~~~~
+
+- LibreRouter
+
+  - LibreRouter (v1)
+
+- Teltonika
+
+  - RUT230 (v1)
+
+
+ath79-nand
+~~~~~~~~~~
+
+- NETGEAR
+
+  - WNDR4300 (v1)
+  
+
+lantiq-xrx200
+~~~~~~~~~~~~~
+
+- Arcadyan
+
+  - o2 Box 6431
+
+
+ramips-mt7621
+~~~~~~~~~~~~~
+
+- D-Link
+
+  - DAP-X1860 (A1)
+
+- GL.iNet
+
+  - GL-MT1300
+  
+- Mercusys
+
+  - MR70X (v1)
+
+- Xiaomi
+
+  - Mi Router 3G
+  
+  
+ramips-mt76x8
+~~~~~~~~~~~~~
+
+- TP-Link
+
+  - RE200 (v3)
+  
+
+realtek-rtl838x
+~~~~~~~~~~~~~~~
+
+- D-Link
+
+  - DGS-1210-10P
+  
+
+ipq40xx-generic
+~~~~~~~~~~~~~~~
+
+- AVM
+
+  - FRITZBox 7520
+  
+  
+ipq40xx-mikrotik
+~~~~~~~~~~~~~~~~
+
+- Mikrotik
+
+  - hAP ac2
+
+
+Bugfixes
+--------
+
+ * Enterasys WS-AP3705i now uses the correct image-name for use with the autoupdater
+   (`#2819 <https://github.com/freifunk-gluon/gluon/issues/2819>`_)
+
+ * Reduce memory Usage for ath10k on ZyXEL WRE6606 devices
+   (`#2842 <https://github.com/freifunk-gluon/gluon/issues/2842>`_)
+
+ * Replace the Workaround for failed boots on ath79 with a proper fix.
+   (`#2784 <https://github.com/freifunk-gluon/gluon/issues/2784#issuecomment-1452126501>`_)
```

Comment:
```suggestion
* Enterasys WS-AP3705i now uses the correct image-name for use with the autoupdater
  (`#2819 <https://github.com/freifunk-gluon/gluon/issues/2819>`_)

* Reduce memory Usage for ath10k on ZyXEL WRE6606 devices
  (`#2842 <https://github.com/freifunk-gluon/gluon/issues/2842>`_)

* Replace the Workaround for failed boots on ath79 with a proper fix
  (`#2784 <https://github.com/freifunk-gluon/gluon/issues/2784#issuecomment-1452126501>`_)
```

Amount of left padding spaces should be a multiple of two, like in the "Known issues" section.